general/chainerrl/baselines/agents/behavioral_cloning.py

```diff
@@ -0,0 +1,128 @@
+"""
+MIT License
+
+Copyright (c) Preferred Networks, Inc.
+"""
+
+from logging import getLogger
+
+import minerl  # noqa: register MineRL envs as Gym envs.
+import numpy as np
+
+import chainer
+from chainer import cuda
+from chainer import functions as F
+
+from chainerrl.agent import AttributeSavingMixin, BatchAgent
+
+logger = getLogger(__name__)
+
+
+class BehavioralCloning(AttributeSavingMixin, BatchAgent):
```

Comment:
maybe using Agent instead of BatchAgent is better since the batch methods aren' implemented, that way you could also delete those methods